### DISCUSSION TITLE
Inverse Problem with DeepXDE for finding a Function instead of Value

### DISCUSSION THREAD

**Discussion by ghost** [lululxvi/deepxde#1823] [Q&A]
Dear Lulu,

I am working with DeepXDE on an inverse problem for my Master’s Thesis, but my problem is a bit different. I have a PDE where I want to solve the inverse problem, but I am looking for a function, not a value. I have a time-dependent PDE with variables \( z \) and \( t \), and I want to find the vector \( D(z) \) dependent on \( z \).

After checking all the documentation, I couldn’t find anything related to what I want. I wanted to ask if it is possible to do what I have in mind with DeepXDE, or if the package is just incompatible with my needs.

I have generated data for the problem, having 4 columns (z, t, M (solution), D (what I want to predict)) in a structured form, if it is useful in any way.

I await your response and thank you for your help.

**Comment by lululxvi** [comment]
https://deepxde.readthedocs.io/en/latest/demos/pinn_inverse/elliptic.inverse.field.html
